totp.go

```diff
@@ -5,10 +5,10 @@ import "time"
 // time-based OTP counters.
 type TOTP struct {
 	OTP
-	interval int
+	interval int64
```

Comment:
I'm open to this, its just that the way I see it, it's that interval is a time period, so should be kept consistent with time. However, as stated before, I'm open to bring it back to `int`

Reply:
I think that we need to add godoc for that property that it is in seconds and let's keep it in `int` type, since it will not break `NewTOTP` functions backward compatibility.